data/bunrodea/bunrodea missions.txt

```diff
@@ -68,7 +68,7 @@ mission "First Contact: Bunrodea (Hostile)"
 				`	(Sit down.)`
 					goto sit
 				`	(Turn around and re-enter my ship.)`
-			`	You reenter your ship, and the aliens don't move to stop you. You decide to turn your ship back on and attempt to take off, but just as you try to activate the repulsor engines, you hear a loud buzzing sound coming from outside. You look out the nearest window, and see that the aliens have moved some sort of vehicle near your ship, buzzing with energy. You attempt to take off, but your ship's repulsor engines aren't moving you anywhere. The vehicle seems like it is deactivating any nearby repulsors.`
+			`	You re-enter your ship, and the aliens don't move to stop you. You decide to turn your ship back on and attempt to take off, but just as you try to activate the repulsor engines, you hear a loud buzzing sound coming from outside. You look out the nearest window, and see that the aliens have moved some sort of vehicle near your ship, buzzing with energy. You attempt to take off, but your ship's repulsor engines aren't moving you anywhere. The vehicle seems like it is deactivating any nearby repulsors.`
```

Comment:
If we're doing re-enter instead of reenter, [Coalition: Pilgramage 5](https://github.com/endless-sky/endless-sky/blob/46a392f85cba1112849ed96a3129aa63e1f36fce/data/coalition/coalition%20missions.txt#L1911) and [FWC Pug 3C](https://github.com/endless-sky/endless-sky/blob/46a392f85cba1112849ed96a3129aa63e1f36fce/data/human/free%20worlds%203%20checkmate.txt#L1603) probably need to be changed, too.